hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDatabase.java

```diff
@@ -280,7 +285,7 @@ private RocksDatabase(File dbFile, RocksDB db, DBOptions dbOptions,
     this.columnFamilies = columnFamilies;
   }
 
-  void close() {
+  public void close() {
```

Comment:
we should add a test to verify that when it is closed, isClosed() is true.

Same for RDBStore